get-involved.md

```diff
@@ -5,20 +5,14 @@ permalink: /get-involved/
 ---
 
 If you're interested in getting involved in the PELUX development, there are
-many ways to help. You can start by getting in touch with us on IRC.
+many ways to help. You can start by getting in touch with us by subscribing to
+our mailing list.
 
-## IRC
+## Public mailing list
 
-Join our IRC channel #PELUX on freenode, where most of the developers hang out.
-join the channel in a web browser:
-<iframe style="border: 0; width: 100%; height: 600px;" src="https://kiwiirc.com/client/irc.freenode.net/#PELUX"></iframe>
+If you have any question regarding PELUX development, subscribe to our mailing list by visiting [mail.pelux.io](https://mail.pelux.io) 
```

Comment:
Better to use neutral language, e.g. any questions regarding PELUX... can be sent to...